### DISCUSSION TITLE
Styles

### DISCUSSION THREAD

**Discussion by mihaeldonko** [microweber/microweber#923] [Q&A]
Is there a way to put new link element in head, that is than active on every page in microweber.

**Comment by engr-sakil** [comment]
Can you explain more details for better understand.What you want.

**Comment by peter-mw** [comment]
Hello,

You can do in your module or template's in `functions.php` file 

```php
event_bind("mw_frontend",function ($params=false) {
    template_head('<script>alert('hello')</script>');;
}); 
```

**Comment by RogerAntonio** [comment]
You could also add the link element in your website's Settings > Advanced > header section.

**Reply by engr-sakil** [reply]
@mihaeldonko you can try this way.

Thanks @RogerAntonio and @peter-mw for yours feedback.
